CHALLENGE.md

```diff
@@ -53,14 +53,14 @@ This is a minimal Create React App SPA that includes:
 
 - Pressing the `EDIT` button opens a browser prompt with the prompt message `New Tournament Name:`, an input field with the current tournament name as pre-filled value and buttons `Cancel` to cancel and `OK` to confirm.
 - The tournament name must contain only Latin letters, numbers, and spaces, not an empty string or only spaces.
-- When `OK` is selected, the tournament name updates immediately in the UI through an "optimistic update", without any loading indicators, and a fetch call to the fake REST API.
+- When `OK` is selected, the tournament name updates immediately in the UI through an "optimistic update" with rollback, without any loading indicators, and a fetch call to the fake REST API.
```

Comment:
Maybe now we've added the rollback bit we can remove this `and a fetch call to the fake REST API` as it kinda sounds like we expect a follow-up call after the request completes, which we don't want them to do 🤔